### ISSUE TITLE
I3PhotonicsService::SelectModuleCoordinate(const I3OMGeo) (Trac #426)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#425]
Currently, the only implementation of this function is:

```text
I3PhotonicsService::SelectModuleCoordinates(double xOM, double yOM, double zOM)
```

Reconstructions typically get the OM position from the I3OMGeoMap in the G-frame.  The function calls to convert from I3OMGeo to (X,Y,Z) can either be in every single reconstruction, or absorbed into the SelectModuleCoordinates function call.

I was able to overload the function in C++ (attached), but got lost in pybindings.  Ideally, this would work in both formats.

Migrated from https://code.icecube.wisc.edu/ticket/426
```json
{
    "status": "closed", 
    "changetime": "2015-03-09T10:07:36", 
    "description": "Currently, the only implementation of this function is:\n{{{\nI3PhotonicsService::SelectModuleCoordinates(double xOM, double yOM, double zOM)\n}}}\n\nReconstructions typically get the OM position from the I3OMGeoMap in the G-frame.  The function calls to convert from I3OMGeo to (X,Y,Z) can either be in every single reconstruction, or absorbed into the SelectModuleCoordinates function call.\n\nI was able to overload the function in C++ (attached), but got lost in pybindings.  Ideally, this would work in both formats.", 
    "reporter": "dunkman", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1425895656480791", 
    "component": "combo reconstruction", 
    "summary": "I3PhotonicsService::SelectModuleCoordinate(const I3OMGeo)", 
    "priority": "normal", 
    "keywords": "photonics-service", 
    "time": "2012-06-28T18:25:40", 
    "milestone": "", 
    "owner": "meike.dewith", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150211T21:16:07: 
* blaufuss changed owner from "nwhitehorn" to "meike"
* blaufuss changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20150211T21:17:48: 
* blaufuss commented:

> Nathan's gone, and don't know the current owner of Photonics...


* blaufuss changed owner from "meike" to "meike.dewith"

**Comment by icecube-trac** [comment]
Trac update at 20150309T10:07:36: 
* jvansanten changed status from "assigned" to "closed"
* jvansanten commented:

> In changeset 130009:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="130009"
> Select module coordinates by I3OMGeo object instead of requiring client code to
> unpack the coordinates itself. This is handy for nefarious
> future-detector-development purposes, but also happens to fix #425.
> ```


* jvansanten changed resolution from "" to "fixed"
